var/spack/repos/builtin/packages/root/package.py

```diff
@@ -28,6 +28,8 @@ class Root(CMakePackage):
     # Development version (when more recent than production).
 
     # Production version
+    version('6.22.04', sha256='a2f066d85db8eb5b5f1c72573923484a6782a47b94954eff64879609f360e951')
```

Comment:
Should we update the latest production version to be preferred?

Reply:
Good question - with ROOT, it's usually wise to stay a bit behind the latest release. I asked around what the status of production ROOT is, and apparently:

> There isn't yet a universally accepted version of ROOT 6.22 . LHCb does not use it but Gaudi tests were failing until 6.22/04. ATLAS seems happy with 6.22/00, which is a bit of a mystery because they also use Gaudi. The latest patch 6.22/06 has still severe I/O issues which impact CMS. Apparently the latters have been understood and a new tag 6.22/08 is expected for the next days.

On the other hand, these issues that are usually edge cases due to the massive complexity, and won't affect most users. Users that care about production stability will most likely select a specific version of root anyway. So no strong opinion what the default should be.

Reply:
No strong opinion from me either.

Latest 6.20 happily coincides with what we currently use.

I see two ways forward:
* Drop the `preferred` label completely and make the latest version the default. People needing specific versions will anyway pin them.
* Mark a "quite stable" version. This seems to be latest 6.20.

Both is fine with me.

Reply:
The ROOT developers would probably more people using the latest versions and reporting bugs. So I'd suggest to drop the `preferred` label.